config/application.rb

```diff
@@ -31,5 +31,7 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    config.assets.debug = true
```

Comment:
perhaps this should go in an environment-specific config file? seems like this could slow you down http://artandlogic.com/2012/12/faster-rails-dev/